posts/migrations/0006_like.py

```diff
@@ -1,24 +0,0 @@
-# Generated by Django 4.0.2 on 2022-03-04 18:27
```

Comment:
Revert this please, this PR shouldn't affect any of the existing migrations. Migrations should only be done when we're updating a model